containers/base/Dockerfile

```diff
@@ -155,18 +155,18 @@ ADD nbconvert /datalab/nbconvert
 RUN ipython profile create default && \
     jupyter notebook --generate-config && \
     if [ -d /datalab/lib/pydatalab/.git ]; then \
-        echo "use local lib"; \
+        echo "use local lib" && \
+        cd /datalab/lib/pydatalab && \
+        /tools/node/bin/npm install -g typescript && \
+        tsc --module amd --noImplicitAny --outdir datalab/notebook/static datalab/notebook/static/*.ts && \
+        /tools/node/bin/npm uninstall -g typescript && \
+        pip install --upgrade-strategy only-if-needed --no-cache-dir . && \
+        rm datalab/notebook/static/*.js; \
       else \
-        git clone https://github.com/googledatalab/pydatalab.git /datalab/lib/pydatalab; \
+        pip install --upgrade-strategy only-if-needed --no-cache-dir datalab==0.1.1701301349; \
```

Comment:
I'm not sure that we should pin the version of the 'datalab' library.

Our dependencies should absolutely be pinned, but for our own library, I worry that we will always want the latest version. If that's the case, then pinning it will just add a manual step we always have to remember to perform.